packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -65,14 +65,11 @@ const bookingsProcedure = authedProcedure.input(commonBookingSchema).use(async (
           OR: [
             /* If user is organizer */
             { userId: ctx.user.id },
-            /* Or part of a collective booking */
+            /* Or part of a team booking */
             {
-              eventType: {
-                schedulingType: SchedulingType.COLLECTIVE,
-                users: {
-                  some: {
-                    id: ctx.user.id,
-                  },
+              user: {
+                some: {
+                  id: ctx.user.id,
```

Comment:
We want bookings to show under a user even if they a part of a round robin booking